lib/Compiler.js

```diff
@@ -129,33 +129,58 @@ class Compiler extends Tapable {
 		this.context = context;
 
 		this.requestShortener = new RequestShortener(context);
+
+		this.running = false;
 	}
 
 	watch(watchOptions, handler) {
+		if (this.running)
+			return handler(
+				new Error(
```

Comment:
Create a new Error class for this error. Inherit from WebpackError. Move the string into this Error class to eliminate duplication